new/LIT6283SaaliLana.xml

```diff
@@ -0,0 +1,55 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6283SaaliLana" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ሰአሊ፡ ለነ፡ ማርያም፡</title>
```

Comment:
Dissimilar to LIT4960Taabbeyo?

Reply:
Thanks Denis for your remark.
Unfortunately I do not have bm:Tasfa1955Saatat handy but, judging from the incipit encoded, the two works seem different.